testing/typing_checks.py

```diff
@@ -29,6 +31,22 @@ def check_parametrize_ids_callable(func) -> None:
     pass
 
 
+# Issue #10999.
+@pytest.mark.skipif(
```

Comment:
This file is not executed, so there is no need for skipping and such.

You can import `TypedDict` from `typing_extensions`, this will work for Python 3.7 as well.